codespell_lib/data/dictionary.txt

```diff
@@ -13348,6 +13411,7 @@ sheduled->scheduled
 shedules->schedules
 sheduling->scheduling
 sheild->shield
+sheme->scheme
```

Comment:
Suggest shame

Reply:
Done "shame".
What about line 8916 "nore->more"?

Reply:
Thanks.

I don't recognise the line number, but `nore->nor, more,`

Reply:
Done